dom.bs

```diff
@@ -6958,8 +6960,14 @@ a boolean <var>delegatesFocus</var>, and a boolean <var>slotAssignment</var>:
   <p>If <var>element</var> is a <a for=Element>shadow host</a>, then:
 
   <ol>
-   <li><p>If <var>element</var>'s <a for=Element>shadow root</a>'s <a for=ShadowRoot>declarative</a>
-   is false, then <a>throw</a> an "{{NotSupportedError!!exception}}" {{DOMException}}.
+   <li><p>Let <var>existing root</var> be <var>element</var>'s <a for=Element>shadow root</a>.
+
+   <li><p>If <var>existing root</var>'s <a for=ShadowRoot>declarative</a>
+   is false, or <var>existing root</var>'s <a for=ShadowRoot>mode</a> does not match
+   <var>mode</var>, or <var>existing root</var>'s <a for=ShadowRoot>delegates focus</a> does
+   not match <var>delegatesFocus</var>, or <var>existing root</var>'s
+   <a for=ShadowRoot>slot assignment</a> does not match <var>slotAssignment</var>,
+   then <a>throw</a> a "{{NotSupportedError!!exception}}" {{DOMException}}.
```

Comment:
I think we should make this a list. Something like:

> If one of the following is true:
>
> 1. _currentShadowRoot_'s declarative is false,
> 2. ...
>
> then ...

I wonder if there's also a place where we can add a comment to not forgot about this when we add a new member.

Speaking of, this does not seem to check clonable?

Reply:
Done.